### ISSUE TITLE
Back Button on Contributor Ftux Not Working

### ISSUE THREAD

**Issue by PixelantDesign** [gitcoinco/web#1685]
Once a Contributor starts the flow, they should be able to click the back button to go back to the landing page that they came from.

https://gitcoin.co/onboard/contributor/

**To Reproduce**

1) Go to https://gitcoin.co/onboard/contributor/
2) Try to click the back button to get back to the landing page.

**Expected behavior**

User should be able to go back.

**Comment by owocki** [comment]
i brought this up more than once during QA of the initial release.

**Comment by thelostone-mc** [comment]
Could you describe when this happens ? 
I can't seem to get it at my end 

![screenshot_20180714-200649](https://user-images.githubusercontent.com/5358146/42725477-912038b2-87a1-11e8-87fc-8129d9609149.jpg)

**Comment by mttmartin** [comment]
I just pushed a commit (https://github.com/mttmartin/web/commit/7453db96a741f99fce7b6e23d77ecea25a12556b) which might fix this specific bug depending its definition.

@PixelantDesign, here are my assumptions regarding the bug as I've understood you:

Current behavior:
1) User gets pointed to https://gitcoin.co/onboard/contributor/
2) When user clicks back button once: they are still on contributor onboard page,  possibly feeling trapped
3) If user clicks a second time: they are returned to their original location

If this is the case, my changes should fix this. 

Here is another issue I see with this, and it might be what you are referring to. The browser back/forward buttons have unintuitive behavior on the onboard page, because their actions differ from the Next/Previous buttons on same page(i.e. they do nothing). Here's an example use-case:

1) Land on contributor onboard page
2) Click Next a couple times
3) User decides they would like to go back to the previous page/step
4) Hit browser back button
4) Page does not change(unless they hit back enough times they hit whatever page in their history that they were on before the contributor onboard)

**Comment by SaptakS** [comment]
@mttmartin I think your commit looks good towards solving this issue. Can you please raise a PR?

**Comment by PixelantDesign** [comment]
Thank you @mttmartin!

where ever user comes from clicking back button should take the user back to the previous page (to avoid trapped feeling).

**Comment by PixelantDesign** [comment]
To receive a tip please update the [July bug list](https://github.com/gitcoinco/web/issues/1671) with your PR, once the task is complete. Thanks!

**Comment by gitcoinbot** [comment]
⚡️ A tip worth 0.04000  ETH (19.27 USD @ $481.66/ETH) has been granted to @mttmartin for this issue from PixelantDesign. ⚡️ 

The sender had the following public comments: 
> Thanks for your help!

Nice work @mttmartin! To redeem your tip, login to Gitcoin at https://gitcoin.co/explorer and select 'Claim Tip' from dropdown menu in the top right, or check your email for a link to the tip redemption page. 

 * $27479.39 in Funded OSS Work Available at: https://gitcoin.co/explorer
 * Incentivize contributions to your repo: <a href='https://gitcoin.co/tip'>Send a Tip</a> or <a href='https://gitcoin.co/funding/new'>Fund a PR</a>
 * No Email? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
